src/tests.rs

```diff
@@ -17,18 +17,29 @@ pub fn setup_tracing() {
 pub fn setup_cache() -> TempDir {
     let tempdir = TempDir::new("tor-cache").expect("create temp dir");
 
-    let consensus = include_str!("test-data/consensus.txt");
-    let microdescriptors = include_str!("test-data/microdescriptors.txt");
-
-    let consensus_path = tempdir.path().join("consensus.txt");
-    let mut consensus_file = File::create(consensus_path).expect("create temp consensus file");
-    write!(consensus_file, "{}", consensus).expect("write temp consensus file");
-
-    let microdescriptors_path = tempdir.path().join("microdescriptors.txt");
-    let mut microdescriptors_file =
-        File::create(microdescriptors_path).expect("create temp microdescriptors file");
-    write!(microdescriptors_file, "{}", microdescriptors)
-        .expect("write temp microdescriptors file");
+    copy(
+        concat!(env!("CARGO_MANIFEST_DIR"), "/tools/directory-cache/authority.txt"),
```

Comment:
To avoid repetition, perhaps create first a var with CARGO_MANIFEST_DIR/tools/directory-cache, and just append the file every time?